app/models/candidate_interface/satisfaction_survey_form.rb

```diff
@@ -4,12 +4,35 @@ class SatisfactionSurveyForm
 
     attr_accessor :question, :answer
 
+    QUESTIONS_WE_ASK = [
+      'I would recommend this service to a friend or colleague',
+      'I found this service unnecessarily complex',
+    ].freeze
+
     validates :question, presence: true
+    validates :question, inclusion: { in: QUESTIONS_WE_ASK, allow_blank: false }
 
```

Comment:
I think we also need `validates :answer, presence: true` and an appropriate message, like “Choose one of the options”

Reply:
As discussed. Candidates should not be forced to give answers to all the questions.